StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetError.swift

```diff
@@ -18,27 +19,29 @@ public enum PaymentSheetError: Error {
     case unknown(debugDescription: String)
     
     /// No payment method types available error.
-    case noPaymentMethodTypesAvailable
+    case noPaymentMethodTypesAvailable(intentPaymentMethods: [STPPaymentMethodType])
 
     /// Localized description of the error
     public var localizedDescription: String {
         return NSError.stp_unexpectedErrorMessage()
     }
 }
 
-extension PaymentSheetError {
+extension PaymentSheetError: CustomDebugStringConvertible {
     /// Returns true if the error is un-fixable; e.g. no amount of retrying or customer action will result in something different
     static func isUnrecoverable(error: Error) -> Bool {
         // TODO: Expired ephemeral key
         return false
     }
     
-    var debugDescription: String {
-        switch self {
-        case .noPaymentMethodTypesAvailable:
-            return "No payment method types available"
-        case .unknown(let debugDescription):
-            return debugDescription
-        }
+    public var debugDescription: String {
+        return "An error ocurred in PaymentSheet. " + {
```

Comment:
Nit: Flagging a typo here (s/ocurred/occurred). Probably want to fix since it's user facing